apps/platform-app/src/app/[locale]/(profile)/(components)/TogglePublishButton/TogglePublishButton.tsx

```diff
@@ -13,6 +13,7 @@ import { useTranslations } from 'next-intl'
 import { publishProfile } from '../../_actions/mutations/publishProfile'
```

Comment:
**suggestion**: Consider using a more descriptive name for the `onClose` function to clarify its purpose.